doc/material-dsp.md

```diff
@@ -32,6 +32,25 @@ This package will be automatically present in the project after [creating a new
 * The package is already present when you select _Material_ theme during project creation:
    ![Selection of Material theme when creating a project using the Uno Template Wizard](assets/material-theme-selection-wizard.png)
 
+It is possible to configure the import process. The _UnoDspImportColors_ item found in the _csproj_ file has a number of attributes we can set:
+
+| Attribute      | Description                     | Default Value           |
+|----------------|---------------------------------|-------------------------|
+| `Include`      | Style files to input            |                         |
+| `Generator`*   | Type of generator to use        | `xaml`                  |
+| `OutputPath`   | Path to use for the output      | Input file directory    |
```

Comment:
are there any restrictions on what values the Include and OutputPath can take? eg: is `json_file.g.cs` a valid OutputPath?

Reply:
cc @carldebilly, @iurycarlos

Reply:
Few points:
1. This `OutputPath` parameter [doesn't exist](https://github.com/unoplatform/uno/blob/master/src/Uno.Sdk/Sdk/Sdk.props.buildschema.json#L288-L297). It will put the generated file right on side of the source one.
2. We should refer to the Tasks' documentation instead of replicating it here.
3. Values for generator are [`Xaml` or `CSharp`](https://github.com/unoplatform/uno/blob/master/src/Uno.Sdk/Sdk/Sdk.props.buildschema.json#L368-L379), not `xaml` and `csharp`.

Reply:
@carldebilly we can simply use the uid: here, no need for an aka link (cc @rajamatt)